ui/src/main/webapp/src/app/components/project-list/project-list.component.scss

```diff
@@ -72,6 +47,14 @@ a {
     .project-info:hover { background-color: rgb(222, 243, 255); }
 }
 
+/* Override Patternfly's middle alignment. */
+.list-view-pf.projects-list .list-group-item.project-info,
+.list-view-pf.projects-list .list-view-pf-description { align-items: flex-start; }
+
+/* Action buttons padding */
+.list-view-pf.projects-list .list-view-pf-actions span { margin-left: 1em; }
+.list-view-pf.projects-list .list-view-pf-actions span a i { font-size: 16pt; }
```

Comment:
I don't agree. Marc and some others demanded quite a lot to have UI like OpenShift. So I think it is better to keep it as it is and then let some UX expert decide what is better.